Jint/Native/Argument/ArgumentsInstance.cs

```diff
@@ -42,9 +42,8 @@ internal ArgumentsInstance(Engine engine) : base(engine, objectClass: "Arguments
             _env = env;
             _strict = strict;
 
-            _properties?.Clear();
-            _intrinsicProperties?.Clear();
-
+            _properties = null;
```

Comment:
Wasn't the goal here to reuse the same instance?

Reply:
The new data type doesn't support Clear, didn't make much of a difference as you can see from perf report. ArgumentsInstances are the big polluters as they were created for function calls, might not have properties materialized that often.

Reply:
You can only say if you run a diff with and without ;)

Reply:
Shouldn't be hard to implement, will add.